utils/Felt.go

```diff
@@ -117,7 +141,7 @@ func StringToByteArrFelt(s string) ([]*felt.Felt, error) {
 
 	arr := r.FindAllString(s, -1)
 	if len(arr) == 0 {
-		return []*felt.Felt{}, fmt.Errorf("invalid string no matches found, s: %s", s)
+		return []*felt.Felt{&felt.Zero, &felt.Zero, &felt.Zero}, nil
```

Comment:
Why does this function now return a valid result instead of an error?

Reply:
Because if "" is a valid shortstring, why wouldn't it be a byte array? No need to return an error, it's just the zero value of the type.
I realized this in an example where starknet.js was hashing correctly the empty string but our code was returning an error just because it was empty.